docker-app/qfieldcloud/core/tests/test_delta.py

```diff
@@ -51,6 +51,24 @@ def setUp(self):
             role=ProjectCollaborator.Roles.ADMIN,
         )
 
+    def tearDown(self):
+        while True:
```

Comment:
Having a timeout is nice idea, I agree.

It is not super required, however...
Since we are using the API to upload/download files, we are now triggering the `process_projectfile` job just after the test is finished. This means a wrapper picks a job from the queue in the end of the job. Since the test framework empties the database, when the worker finishes and the wrapper tries to update the database, it results in an error. So to prevent a bunch of confusing errors in the logs.